README.md

```diff
@@ -1 +1,26 @@
-# community
\ No newline at end of file
+# Human Cell Atlas Data Coordination Platform community
+
+** Work in Progress **
+
+Welcome to the community!
+
+## Communicating
+
+To get started, [join the HumanCellAtlas Slack workspace](http://join-slack.humancellatlas.org/) to receive an invitation. 
+
+### Browse and join Slack channels:
+
+  - #general is for
+  - #dcp is for
+  - #ingestion-service is for
+  - #data-store is for
+  - #secondary analysis is for
+  - #data-portal is for
+  - ...
+
+## Community Governance
+### Oversight
+### Decision Making, Conflict Resolution, and Facilitation
+### Code of Conduct
+### Project ownership and the charter process
```

Comment:
I'm trying to think of an example of a project in the DCP. As far as I can think of, we are all making products. Do you have an example in mind?